samples/AuthRemoteFormsAuth/AuthRemoteFormsAuthAppHost/Program.cs

```diff
@@ -0,0 +1,17 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var remoteApiKey = builder.AddParameter("apiKey", Guid.NewGuid().ToString(), secret: true);
+
+var frameworkApp = builder.AddIISExpress("iis")
```

Comment:
TIL about these extensions! Preety neat!

Reply:
Yes - thanks to some great work by @CZEMacLeod!